### ISSUE TITLE
[Usability issues] color contrast from checkbox terms is blocking some users to move forward

### ISSUE THREAD

**Issue by eduardaloureiro** [status-im/status-react#17779]
# Bug Report
The checkbox from terms and condition initial screen inside the app has not enough color contrast and users using the mobile with lower lightning rate are not able to see this, getting block into the screen unless they tap on it by instinct.

## Problem 
[comment]: # (A problem description)
  Users cannot see the checkbox from terms and condition to move forward.
  
#### Expected behavior
Change checkbox color contrast to at least 4.5:1 ratio so users can see it clearly and move forward into the app.

#### Actual behavior
Color contrast lower than 4:5:1 ration so users cannot see unless they have the lightning rate at max.

### Reproduction

1. Put your device screen lightning medium to low
2. open the app for the first time > users cant see the checkbox b/c the lower contrast ration with the background vs checkbox color.

### Additional Information
 https://accessibleweb.com/color-contrast-checker/

- Status version: nightly 
- Operating System: Android, iOS - both.

[comment]: # (Please, add logs/notes if necessary)

**Comment by churik** [comment]
@eduardaloureiro thank you for the issue!
@Francesca-G can you please check it, as I have no opinion on the matter.


For me the checkbox is visible
![IMAGE 2024-06-21 13:50:54](https://github.com/status-im/status-mobile/assets/4557972/22183500-48ae-4128-ab83-1bde3f1f3868)

**Comment by Francesca-G** [comment]
> @eduardaloureiro thank you for the issue! @Francesca-G can you please check it, as I have no opinion on the matter.
> 
> For me the checkbox is visible

I agree, I even turned down all the way screen brightness and activated the night shift mode on my iPhone and it's still visible. It has a lower contrast compared to the white text but the text is clearly readable and the checkbox is right next to it so I don't see it being an issue.

Maybe it's something to be considered on some old Android screens? I don't see any point in changing it as far as I can test.

**Comment by eduardaloureiro** [comment]
Hey Francesca!
Its almost an year since I left this here, so considered my report deprecated. 

Best, 
Eduarda
